tikv-configuration-file.md

```diff
@@ -312,6 +312,35 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 + Default value: 45% of the size of total system memory
 + Unit: KB|MB|GB
 
+## storage.flow-control
+
+Configuration items related to the flow control mechanism in TiKV. This mechanism supersedes the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck Raftstore or Apply threads when the write traffic is high.
```

Comment:
```suggestion
Configuration items related to the flow control mechanism in TiKV. This mechanism supersedes the write stall mechanism in RocksDB and controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck of Raftstore or Apply threads when the write traffic is high.
```
missing `of`?

Reply:
No. "stuck" is an adjective that modifies Raftstore or Apply.